website/pages/en/network/delegating.mdx

```diff
@@ -41,64 +55,89 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
 
 ### Choosing a trustworthy Indexer with a fair reward payout for Delegators
 
-This is an important aspect to understand. First, let's discuss three very important values, which are the Delegation Parameters.
+In order to understand how to choose a trustworthy Indexer, you need to understand the Delegation Parameters.
 
-Indexing Reward Cut - The indexing reward cut is the portion of the rewards that the Indexer will keep for themselves. This means that if an Indexer's rewards are set to 100%, as a Delegator you will get 0 indexing rewards. If you see it set at 80% in the UI, that means as a Delegator, you will receive 20%. An important note - at the beginning of the network, Indexing Rewards will account for the majority of the rewards.
+#### Delegation Parameters
+
+- **Indexing Reward Cut** - The portion of the rewards the Indexer will keep for themselves.
+  - If an Indexer's reward cut is set to 100%, as a Delegator, you will get 0 indexing rewards.
```

Comment:
```suggestion
  - If an Indexer's reward cut is set to 100%, as a Delegator, you will not get any indexing rewards.
```

Reply:
-I like the numbers here--make it more direct